sensu_configs/init.d/sensu-service

```diff
@@ -197,6 +197,12 @@ wait_for_stop() {
     return $stopped
 }
 
+deregister_client() {
+    log_action_msg "Deregistering sensu-client"
+    echo "{ \"name\": \"api_call\", \"output\": \"delete\", \"process\": \"init\", \"user\": \"root\", \"status\": 2, \"handler\": \"${CLIENT_DEREGISTER_HANDLER}\"}" > \
```

Comment:
Can we make this a warning (`1`)? As the event will temporarily display via the API/Dashboard.